.gitignore

```diff
@@ -105,4 +105,5 @@ hs_err_pid*
 .vscode/
 
 # Python Virtual Env
-.venv
\ No newline at end of file
+.venv
+venv
```

Comment:
Unnecessary change

Reply:
I disagree. Some people (including myself) use venv instead of .venv and it's a good QOL addition to not have to sift through the venv to find your changes. 

That being said, is there any security issues or genuine concerns that arise having this `venv` directory ignored?

Reply:
I agree with @cccs-cat001 on this topic, but I'd propose to make this change in a separate PR to avoid mixing QOL changes with feature changes 😉

Reply:
I'm ok with this... just proposing a faster (IMHO) path :slightly_smiling_face: